### PR TITLE
[HAC-85] As a developer I have RWD website without grommet.io

### PR DESCRIPTION
**Pull request by kkotelczuk**
- Remove Grommet boxes from app.react.jsx
- added .scss loader for flexbox-grid
- added RWD layout using react-flexbox-grid

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,9 +22,11 @@
     "react": "^0.14.0",
     "react-dom": "^0.14.0",
     "react-dropzone": "2.2.3",
+    "react-flexbox-grid": "^0.9.4",
     "react-inline-grid": "^0.5.3",
     "react-inlinesvg": "^0.4.2",
     "react-onclickoutside": "^4.0.1",
+    "react-responsive": "^1.1.1",
```
Comment (patrykkopycinski) [inline package.json]: is it necessary?

Comment (kkotelczuk) [inline package.json]: No, it will be removed.

Comment (patrykkopycinski) [inline package.json]: :+1:

**Comment by kkkowalski** [comment]
lgtm

**Comment by mkucharz** [comment]
Lgtm

**Pull request merged**